core-ui/src/components/Functions/FunctionDetails/Tabs/Code/CodeAndDependencies/CodeAndDependencies.js

```diff
@@ -72,6 +72,7 @@ export default function CodeAndDependencies({ func }) {
     }
 
     setDisabledCause(DISABLED_CAUSES.VALID);
+    setDependencies(deps);
```

Comment:
When is it needed? I removed it and it works just fine.

Reply:
It's needed when dependencies input from UI contains only spaces, `\n` or `\t`. If you would pass only these symbols in the dependencies on the UI, then the function would report a build error.